CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2.3.0 (2024-07-24)
```

Comment:
```suggestion
## UNRELEASED
```

We will set the version and the date when we will release the module

Reply:
Done!